ERCS/erc-7087.md

```diff
@@ -0,0 +1,163 @@
+---
+eip: 7087
+title: New features for Web3 URL in Auto Mode
+description: Add features to Web3 URL in auto mode
+author: Qi Zhou (@qizhou), Nicolas Deschildre (@nand2)
+discussions-to: https://ethereum-magicians.org/t/erc-7087-mime-type-for-web3-url-in-auto-mode/14471
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-28
+requires: 6860
+---
+
+## Abstract
+
+This standard extends the [ERC-6860](./eip-6860.md) `web3://` standard in auto mode with new features :
+
+- Determine the MIME type of a URL via additional query parameters.
+- Ability to name fields when ``?returns=`` is used.
```

Comment:
Your abstract is a bit too terse. You can expand a bit on how this proposal is implemented. The reader should get a high level technical understanding of the proposal just by reading the abstract.